libvast/src/command.cpp

```diff
@@ -352,6 +372,10 @@ parse(const command& root, command::argument_iterator first,
 
 bool init_config(caf::actor_system_config& cfg, const command::invocation& from,
                  std::ostream& error_output) {
+  // Adjust default file verbosity for the start command.
+  if (from.name() == "start")
```

Comment:
Tbh, after working on this I'm not even sure that we really need this special case: It seems perfectly reasonable to say that `vast` will not write logs to a file unless log-file and/or file-verbosity are specified, including `vast start`.

Reply:
You are right, it is more customary not to write logfiles unless specified. But think about the usual scenario from a users perspective: You normally don't think about enabling the log file until the moment something goes wrong and you need them, on the other hand, verbose logging is still not that high volume that is poses a problem, so it does not hurt to just turn them on from the start.

Reply:
> You normally don't think about enabling the log file until the moment something goes wrong and you need them

This hits the nail on the head I think—it's exactly what prompted us to create the verbose log level just recently. I think having it enabled by default is fine.

Reply:
I'd argue that this is the *developers* perspective rather than the users perspective. A user toying around with vast will most likely not dig through logs to figure out an error. (and if the error is non-repeatable, he will most likely just shrug it off and move on) And creating a log directory does hurt: the original reason for this PR was user complaints after all.

Maybe a better solution to this problem would be to set the default console verbosity to "debug" for non-release builds? Then you could not forget to turn on verbose logs until it is too late.

Reply:
As discussed in slack, I moved the default location of the log file inside `vast.db/`, this way users don't notice it but developers can still look at it if they want to.